src/app/components/applications/SSV/MyAccount/components/Withdraw/components/ValidatorFlow/ValidatorFlow.tsx

```diff
@@ -26,9 +27,10 @@ const ValidatorFlow = () => {
   const process: SingleCluster = processStore.getProcess;
   const cluster = process.item;
   const clusterBalance = walletStore.fromWei(cluster.balance);
-  const [inputValue, setInputValue] = useState<number | string>('');
   const applicationStore: ApplicationStore = stores.Application;
   const [userAgree, setUserAgreement] = useState(false);
+  const { termsConditionWrapper, checkedCondition } = useTermsAndConditions();
+  const [inputValue, setInputValue] = useState<number | string>('');
```

Comment:
From this variable name it's not clear what it is used for

Reply:
done